researchstack-mpower/src/main/java/org/sagebionetworks/research/mpower/researchstack/inject/MPowerResearchStackModule.java

```diff
@@ -37,6 +36,14 @@ static MpTaskFactory provideMpTaskFactory() {
     static ResearchStack provideResearchStack(Context context) {
         MpResearchStack researchStack = new MpResearchStack(context);
         ResearchStack.init(context, researchStack);
+
+        // We don't use a pin code for MPower, so just plug one in for the app to always use
+        if (StorageAccess.getInstance().hasPinCode(context)) {

```

Comment:
I never really liked this approach on BP.  I think we should enabled no pin code at a lower ResearchStack level.  Seems silly to do this fake pin code stuff.